_dev/back/back.scss

```
@@ -302,6 +302,15 @@
     width: 35px;
 }
 
+@media (max-width:768px) {
+    .listing-row.row {
+        padding-top: 9px;
+    }
+    .listing-row div {
+        padding: 9px 0;
```

```suggestion
        padding-top: 0.5625rem;
    }
    .listing-row div {
        padding: 0.5625rem 0;
```

Using rem is always a good idea but I think it's nitpick there